### PR TITLE
Update sentence_translations.json

### PR DESCRIPTION
**Pull request by cheskoxd**
- Fixed grammatical errors
- Swapped some words to make them sound more native (there are words in Spanish that are not commonly used, so I just changed them to the more common ones) 
- Fixed some sentences that were broken due to the translation  
- Updated some sentences that didn't make sense in Spanish

I read this out loud multiple times, to make sure it doesn't sound like it was translated.

### CODE REVIEW

File: 2023/convolutions2/spanish/sentence_translations.json [hunk 1]
```diff
@@ -11,7 +11,7 @@
  {
   "input": "Suppose I take a normal distribution with this familiar bell curve shape, and I have a random variable x that's drawn from that distribution.",
   "model": "nmt",
-  "translatedText": "Supongamos que tomo una distribución normal con esta familiar forma de curva de campana y tengo una variable aleatoria x que se extrae de esa distribución.",
+  "translatedText": "Supongamos que tomo una distribución normal con esta familiar curva en format de campana y tengo una variable aleatoria x que se extrae de esa distribución.",
```
Comment (Yag000) [inline 2023/convolutions2/spanish/sentence_translations.json:14]:
```suggestion
  "translatedText": "Supongamos que tomo una distribución normal con esta familiar curva en forma de campana y tengo una variable aleatoria x que se extrae de esa distribución.",
```

**Review by cheskoxd — COMMENTED** [review]
fixed typo

File: 2023/convolutions2/spanish/sentence_translations.json [hunk 2]
```diff
@@ -533,7 +533,7 @@
  {
   "input": "Some of you might like to think of this as a kind of dot product.",
   "model": "nmt",
-  "translatedText": "A algunos de ustedes les gustaría pensar en esto como una especie de producto escalar.",
+  "translatedText": "A algunos de ustedes les gustara pensar en esto como una especie de producto escalar.",
```
Comment (Yag000) [inline 2023/convolutions2/spanish/sentence_translations.json:536]:
```suggestion
  "translatedText": "A algunos de ustedes les gustará pensar en esto como una especie de producto escalar.",
``` Small typo

**Review by Yag000 — APPROVED** [review]

**Comment by 3b1b** [comment]
Thanks so much, I appreciate you taking the time.

**Comment by cheskoxd** [comment]
> Thanks so much, I appreciate you taking the time.

Very welcome!

**Pull request merged**